src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -511,6 +511,9 @@ private void InstrumentType(TypeDefinition type)
         private void InstrumentMethod(MethodDefinition method)
         {
             var sourceFile = method.DebugInformation.SequencePoints.Select(s => _sourceRootTranslator.ResolveFilePath(s.Document.Url)).FirstOrDefault();
+
+            if (string.IsNullOrEmpty(sourceFile)) return;
```

Comment:
Can you add an information log with method/module information

Reply:
Done